README.md

```diff
@@ -25,6 +25,8 @@ CreatorKit project is spread across couple of organizations :-
     * [Webscripts](https://github.com/CreatorKit/webscripts): Package which provides web interface for provisioning CreatorKit devices. It includes webpages as well as webservices.
     * [Libobjects](https://github.com/CreatorKit/libobjects): Collection of lwm2m and custom ipso objects for Clicker apps.
     * [Button Gateway](https://github.com/CreatorKit/button-gateway): CreatorKit [Project 0](projects/Project_0.md) application for Creator Ci40 platform
+    * [Led Controller](https://github.com/CreatorKit/led-controller): CreatorKit [Project 1](projects/Project_1.md) application for Creator Ci40 platform
```

Comment:
Please add LED controller in a separate commit, so that it can be merged separately to create a new tag 0.9.1 for release that includes LED Controller project.

This PR can then later be merged for 0.9.2 tag to add Light Controller project